cli/bpmetadata/tfconfig_test.go

```diff
@@ -368,3 +368,28 @@ func TestTFIncompleteProviderVersions(t *testing.T) {
 		})
 	}
 }
+
+func TestTFVariableSortOrder(t *testing.T) {
+	tests := []struct {
+		name         string
+		configPath   string
+		expectOrders map[string]int
+	}{
+		{
```

Comment:
Could we add a few more cases with no outputs and invalid output block declaration?

Reply:
Done